tests/files/woningbouwplannen.json

```diff
@@ -45,6 +45,14 @@
               }
             },
             "description": "Buurten waarin het woningbouwplan ligt"
+          },
+          "buurtenAsScalar": {
+            "type": "array",
+            "relation": "gebieden:buurten",
+            "items": {
+              "type": "string"
+            },
+            "description": "Buurten waarin het woningbouwplan ligt, simply as scalar"
           }
```

Comment:
gebruiken buurten en buurtenAsScalar hier dezelfde onderliggende data? Of moet er een duplicate kolom worden toegevoegd aan de tabel? Ik had begrepen dat het de bedoeling was dat deze array van loose identifiers by default uit de api zou moeten komen, zonder explicitiet te moeten noemen in het schema. Maar hier is ook iets voor te zeggen.